release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -150,6 +150,13 @@ With this release, you can now require your workloads to use a specific security
 
 For more information, see xref:../authentication/managing-security-context-constraints.adoc#security-context-constraints-requiring_configuring-internal-oauth[Configuring a workload to require a specific SCC].
 
+[id="ocp-4-14-auth-psa-exemptions"]
+==== Namespaces exempt from pod security admission
+
+With this release, the following system namespaces are exempt from pod security admission control: `default`, `kube-public`, `kube-system`.
```

Comment:
Would listing thse items as bullet points be better? The last item in the running list is missing "and" before it.

Reply:
I did use bullet points for my main content PR for this, but I usually try to be more concise in the release notes. I'll think on it whether to make these bullets, or to add the "and". Thanks!